packages/customize-widgets/src/index.js

```diff
@@ -31,12 +30,6 @@ export function initialize( editorName, blockEditorSettings ) {
 	);
 	registerCoreBlocks( coreBlocks );
 
-	if ( process.env.GUTENBERG_PHASE === 2 ) {
-		__experimentalRegisterExperimentalCoreBlocks( {
-			enableLegacyWidgetBlock: true,
-		} );
-	}
```

Comment:
Do we still want to register other blocks like Navigation block?

Reply:
Oh yeah, oops, we do 😀 👍